openff/toolkit/tests/test_molecule.py

```diff
@@ -1799,10 +1799,10 @@ def assert_check():
 
         assert_check()
         assert qcschema.extras["test_tag"] == "test"
-        # now run again when no extras
-        qcschema = ethanol.to_qcschema()
-        assert_check()
-        assert qcschema.extras is None
+        # # now run again when no extras
+        # qcschema = ethanol.to_qcschema()
+        # assert_check()
+        # assert qcschema.extras is None
```

Comment:
@pavankum Could you take a pass at this and either delete it, or change it to test some important part of the new functionality?

Reply:
resolved by modifying old test to check for canonical smiles in extras entries